packages/cli/src/cli.ts

```diff
@@ -93,5 +110,7 @@ function parseOptions(argv: readonly string[]): CLIArguments {
     .command('ci', 'start a server and run the test suite', (yargs) => runOptions(startOptions(yargs)))
     .demandCommand()
     .help()
-    .parse(argv) as unknown as CLIArguments // types generated by yargs are inadequate
+    .parse(argv)
+
+  return { command: parsed._[0], ...parsed } as unknown as Options // types generated by yargs are inadequate
```

Comment:
TypeScript is unable to narrow the enum types when looking at the array value, which is why I've extraced it into a property here so the types are properly narrowed in the `if` statements above.